packages/auth/package.json

```diff
@@ -61,6 +70,10 @@
         "require": "./dist/browser-cjs/internal.js",
         "import": "./dist/esm2017/internal.js"
       },
+      "extension": {
+        "require": "./dist/extension-cjs/internal.js",
+        "import": "./dist/extension-esm2017/internal.js"
+      },
```

Comment:
A few lines down in the `files` property, need to add "extension/package.json".

Reply:
Done.